src/home/components/home-banner/HomeBanner.test.tsx

```diff
@@ -0,0 +1,77 @@
+import { cleanup } from '@testing-library/react';
```

Comment:
[nit]: In accordance with how we already pass `renderHook`, `waitFor` and `within`, I suggest passing `cleanup` via [setupTest.js](https://github.com/raccoongang/frontend-app-catalog/blob/rg-tick/develop/src/setupTest.js#L6) file